projects/solv-protocol/index.js

```diff
@@ -0,0 +1,75 @@
+const sdk = require('@defillama/sdk');
+const axios = require('axios')
+const { transformBscAddress, transformEthereumAddress, transformPolygonAddress, transformArbitrumAddress } = require("../helper/portedTokens");
+
+// token list
+const tokenListsApiEndpoint = "https://token-list.solv.finance/vouchers-prod.json"
+
+const ethereumTVL = async (timestamp, block, chainBlocks) => {
+    const transform = await transformEthereumAddress();
+    return tvl(timestamp, block, chainBlocks, "ethereum", 1, transform);
+}
+
+const polygonTVL = async (timestamp, block, chainBlocks) => {
+    const transform = await transformPolygonAddress();
+    return tvl(timestamp, block, chainBlocks, "polygon", 137, transform);
+}
+
+const arbitrumTVL = async (timestamp, block, chainBlocks) => {
+    const transform = await transformArbitrumAddress();
+    return tvl(timestamp, block, chainBlocks, "arbitrum", 42161, transform);
+};
+
+const bscTVL = async (timestamp, block, chainBlocks) => {
+    const transform = await transformBscAddress();
+    return tvl(timestamp, block, chainBlocks, "bsc", 56, transform);
+};
+
+async function tvl(timestamp, block, chainBlocks, network, chainId, transform) {
+    let balances = {}; // Setup the balances object
+    const tokens = await tokenList(chainId);
+    console.log(tokens)
```

Comment:
the PR looks good, but can you remove the `console.log` after debugging/testing
instead of fetching one token balance at a time, can you replace it with `sumTokens`  method, the change would be for loop with

    await sumTokens(balances, tokens.map(t => [t.address, t.pool]), chainBlocks[network], network)
    
  this is bit faster as we make batch calls instead of making sequential calls

Reply:
done